apps/web/src/components/(dashboard)/settings/layout/desktop-nav.tsx

```diff
@@ -1,11 +1,11 @@
 'use client';
 
-import { HTMLAttributes } from 'react';
+import type { HTMLAttributes } from 'react';
```

Comment:
Automating formatting on save does some things like this automatically...

Reply:
This is expected, we added some lint rules but didn't want to reformat the entire codebase and add to confusion so it's a progressive kind of thing